docs/Reference/API.md

```diff
@@ -48,9 +48,9 @@ There should only be a single store in your app.
 * `reducer: Reducer`: Required. A [reducer](./Glossary.md#reducer) function that returns the next state tree, given
 the current state tree and the action to handle.
 
-* `initialState: any`: The initial [state](./Glossary.md#state). You may optionally specify it
+* `initialState: Object`: The initial [state](./Glossary.md#state). You may optionally specify it
 to hydrate the state from the server in universal apps, or to restore a
-previously serialized user session.
+previously serialized user session. The object keys must match the reducer names managed by `combineReducers`.

```

Comment:
This is a tricky one. Please don't *have to* use `combinedReducers`. They may be passing a single reducer function that operates on an Immutable tree. Maybe rephrasing it as “If you use `combineReducers` to produce the root reducer function, this must be an object with the same shape as `combineReducers` keys.” would help.